.travis/pulp_file-tests.sh

```diff
@@ -2,11 +2,14 @@
 # coding=utf-8
 
 # From the pulp-server/pulp-api config-map
-echo "machine localhost
+echo "machine $(hostname)
```

Comment:
We could also just do `default` for the entire line. It matches every host. And since we're replacing the entire file (rather than appending to it with >>), it pre-empts any possible changes in the future.

Reply:
i want to keep it as is. however, i am open to changing this in a follow up PR.